compiler/ballerina-lang/src/main/java/org/ballerinalang/util/diagnostic/DiagnosticErrorCode.java

```diff
@@ -505,7 +505,7 @@ public enum DiagnosticErrorCode implements DiagnosticCode {
     TABLE_KEY_SPECIFIER_MISMATCH("BCE3301", "table.key.specifier.mismatch"),
     KEY_SPECIFIER_SIZE_MISMATCH_WITH_KEY_CONSTRAINT("BCE3302", "key.specifier.size.mismatch.with.key.constraint"),
     KEY_SPECIFIER_MISMATCH_WITH_KEY_CONSTRAINT("BCE3303", "key.specifier.mismatch.with.key.constraint"),
-    MEMBER_ACCESS_NOT_SUPPORT_FOR_KEYLESS_TABLE("BCE3305", "member.access.not.supported.keyless.table"),
+    MEMBER_ACCESS_NOT_SUPPORT_FOR_TABLE("BCE3305", "member.access.not.supported.table"),
```

Comment:
Also, when updating the error codes, we usually use the dotted version of the same enum member name as the message key.
```suggestion
    MEMBER_ACCESS_NOT_SUPPORTED_FOR_TABLE("BCE3305", "member.access.not.supported.for.table"),
```

Reply:
Updated d75e529ac2f2918cc6b8d4ea8d6f0b8d5804d001